### PR TITLE
51 prepare release 07

### PR DESCRIPTION
**Pull request by stojanov-igor**
Final preparation before release 0.7

Contains 
- updated insource documentation
- revised tests
- regenerated weights
- small fixes

Budget: $20

### CODE REVIEW

File: pallets/dao/src/lib.rs [hunk 1]
```diff
@@ -43,26 +45,75 @@
...
-//!
-//! - `remove_tasks` - Function used for a visionary to remove tasks from his organization.
+//!		Inputs:
+//! 		- org_id: Hash
+//! 
+//! Storage Items:
+//! 	Vision: Vision document 
+//! 	VisionCount: Number of total visions in the system
+//! 	Organizations: List of all organizations in the system
+//!     OrganizationCount: Total numbers of organizations in the system
```
Comment (vivekvpandya) [inline pallets/dao/src/lib.rs:111]: Seems like extra space(s) at beginning of line.

File: pallets/profile/src/benchmarking.rs [hunk 2]
```diff
@@ -33,6 +33,7 @@ fn assert_last_event<T: Config>(generic_event: <T as Config>::Event) {
 	frame_system::Pallet::<T>::assert_last_event(generic_event.into());
 }
 
+
```
Comment (vivekvpandya) [inline pallets/profile/src/benchmarking.rs:36]: Is space required?

**Review by vivekvpandya — APPROVED** [review]
LGTM, Just a suggestion: if you specify inputs in doc comments then output can also be shown where applicable.

**Comment by stojanov-igor** [comment]
Minor space issues can be resolved in the future. Merging this into `develop`.

**Pull request merged**